src/storage.ts

```diff
@@ -393,6 +420,12 @@ export class Storage extends Service {
    * @param {StorageOptions} [options] Configuration options.
    */
   constructor(options: StorageOptions = {}) {
+    const AUTO_RETRY_DEFAULT = true;
```

Comment:
Would it make sense to move these constants to the class level along with a comment for consistency with other files? For example https://github.com/googleapis/nodejs-storage/blob/fbb90c2b5e01c971c1aa0bbc54ddabb2df89673d/src/bucket.ts#L381

Reply:
fixed! good callout